### DISCUSSION TITLE
osu lazer crashes on click in main menu

### DISCUSSION THREAD

**Discussion by ROKOLYT** [ppy/osu#18244] [Q&A] [answered]
### Type

Crash to desktop

### **Summary:**
+ osu lazer crashes on MAIN MENU
+ game works till I click ANYTHING in main menu
+ opening osu stable and running it in the bg fixes the crash
+ when I'm in song selection menu I can safely close osu stable w/o crashes (game works perfectly except for main menu)
+ returning to main menu w/o osu stable running and clicking anything will cause a crash
+ The crash also happened when I didn't have osu stable installed
+ I had the exact same crash like 2 months ago but thought that it would be fixed (and it fixed itself? now it's back? donno?)

**Specs:**
+ Win 11 (Beta 22621.1)
+ i5 9400F
+ GTX 1060 6GB
+ 16 GB RAM
+ Nvidia game ready driver ver. 512.59

### Screenshots or videos

![image](https://user-images.githubusercontent.com/98853508/168101250-1300a03f-cafb-473e-8b0e-4b64269666f2.png)
![image](https://user-images.githubusercontent.com/98853508/168101410-ae169df7-36dd-44cc-8622-62a3e38123a0.png)

### Version

2022.509.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/8680028/database.log)
[network.log](https://github.com/ppy/osu/files/8680029/network.log)
[performance.log](https://github.com/ppy/osu/files/8680030/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8680031/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8680033/updater.log)

**Comment by peppy** [comment, marked as answer]
It's a display driver crash. Please update your graphics drivers.

**Reply by ROKOLYT** [reply]
I updated my graphics driver to 512.77 (newest version) and the issue persists. It changed a little bit though, now it crashes w/o clicking anything. Opening osu stable and running it in the background still fixes the issue. windows errors are still the same.

**Reply by bdach** [reply]
If you have shadowplay enabled please try disabling it.

**Reply by ROKOLYT** [reply]
@bdach It worked thanks.
